src/vl-multiselect.js

```diff
@@ -24,7 +24,15 @@ export class VlMultiSelect extends VlSelect {
 
   connectedCallback() {
     this.classList.add('vl-multiselect');
+    this.setAttribute('multiple', 'multiple');
     this.setAttribute('data-vl-multiselect', '');
+    const options = Array.from(this.querySelectorAll('option'));
+
+    const hasSelected = options.some((option) => {
+      return option.hasAttribute('selected') === true;
+    });
```

Comment:
Ik zou deze logica in een aparte functie `_hasSelected` steken zodat de `connectedCallback` minder logica bevat en daardoor leesbaarder wordt.

Reply:
Fixed.